modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-localnet-nad-cli_{context}"]
+= Creating a NAD for localnet topology using the CLI
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the underlying physical network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Kubernetes NMState Operator.
+* You have created a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge.
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` object:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: localnet-network
+  namespace: default
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1", <1>
+            "name": "localnet-network", <2>
+            "type": "ovn-k8s-cni-overlay", <3>
+            "topology": "localnet", <4>
+            "netAttachDefName": "default/localnet-network" <5>
+    }
+----
+<1> The CNI specification version. The required value is `0.3.1`.
+<2> The name of the network. This attribute must match the value of the `spec.desiredState.ovn.bridge-mappings.localnet` field of the `NodeNetworkConfigurationPolicy` object that defines the OVS bridge mapping.
```

Comment:
Spell out OVS.

This term and CNI are spelled out in the next document.

Reply:
OVS is spelled out in the Prerequisites. Both CNI and OVS are also spelled out in the assembly intro.